apps/studio/src/plugins/CustomTabulatorModule.ts

```diff
@@ -0,0 +1,19 @@
+import { Module } from "tabulator-tables";
+
+export class CustomTabulatorModule extends Module {
```

Comment:
Hmm, I think this probably needs  better name? Maybe `HeaderSortTabulatorModule`, or something?

Reply:
I've still got one very little issue. This is how it looks if someone open a table that they have opened before:

![image](https://github.com/beekeeper-studio/beekeeper-studio/assets/38707148/34f05934-cd8b-48f6-a485-636d0e935c97)

It's using the previous width sizes, and we're adding a new element to the header. It's easy to adjust though, just right click > `resize all column to match contents`.

Reply:
It's good if someone never opened the table before. It looks like the gif I showed you.